### PR TITLE
update .htaccess for rdf content negotiation

### PR DESCRIPTION
**Pull request by kabulkurniawan**
thanks :)

### CODE REVIEW

File: sepses/.htaccess [hunk 1]
```diff
@@ -1,4 +1,36 @@
...
 RewriteEngine on
+
+AddType application/rdf+xml .rdf
+AddType application/rdf+xml .owl
+AddType text/turtle .ttl
+AddType application/n-triples .n3
+AddType application/ld+json .json
+
 RewriteRule ^$ http://sepses.ifs.tuwien.ac.at [R=302,L]
 RewriteRule ^(.*)$ http://sepses.ifs.tuwien.ac.at/$1 [R=302,L]
```
Comment (davidlehn) [inline sepses/.htaccess]: Rules are checked in order, so this wildcard rule will catch everything.  You should move it to the bottom of the file.

Comment (kabulkurniawan) [reply sepses/.htaccess]: done.. thankyou

Comment (kabulkurniawan) [reply sepses/.htaccess]: Hi I still got a problem when I open this link  http://w3id.org/sepses/vocab/ref/cve/ontology.ttl via protege, the error message is :
sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target

but when I directly open this link: http://sepses.ifs.tuwien.ac.at/vocab/ref/cve/ontology.ttl, it works fine.

do you see that this is a problem from w3id?

**Comment by dgarijo** [comment]
This is was Protege problem.
They have fixed it in the Protege version 5.5. I just opened
https://w3id.org/sepses/vocab/ref/cve/ontology.ttl fine.

**Comment by AnnaWagner** [comment]
I just updated the htaccess of BOT while I was fixing my htaccess files, should work fine now. Just noticed during copying the rewrite rules that BOT is not provided in any serialisation but TTL, maybe this could be supplemented by RDF, JSON-LD and NT?

**Pull request merged**